shared/wallets/wallet-list/index.js

```diff
@@ -97,4 +112,34 @@ const AddWallet = FloatingMenuParentHOC(_AddWallet)
 
 const rowHeight = isMobile ? 56 : 48
 
-export {Wallet, AddWallet}
+type Props = {
+  wallets: Array<{
+    accountID: AccountID,
+    name: string,
+    keybaseUser: string,
+    contents: string,
+  }>,
+  selectedAccount: ?AccountID,
+  onSelectAccount: (accountID: AccountID) => void,
+  onAddNew: () => void,
+  onLinkExisting: () => void,
+}
+
+const WalletList = (props: Props) => (
+  <Box2 direction="vertical" style={{height: '100%', width: 240}}>
+    {props.wallets.map(w => (
+      <Wallet
```

Comment:
lists should
1. Use <List>
2. just pass the id and have the wallet be connected and get this stuff
3. Be in their own component file